### PR TITLE
Switch to newer GPU CI images

### PR DESCRIPTION
**Pull request by charlesbluca**
- [ ] Closes #xxxx
- [ ] Tests added / passed
- [ ] Passes `pre-commit run --all-files`

### CODE REVIEW

**Comment by charlesbluca** [comment]
rerun tests

**Comment by charlesbluca** [comment]
rerun tests

**Comment by charlesbluca** [comment]
rerun tests

File: continuous_integration/gpuci/build.sh [hunk 1]
```diff
@@ -27,32 +27,37 @@ export CUDA_REL=${CUDA_VERSION%.*}
...
+rapids-logger "Check GPU usage"
 nvidia-smi
 
-gpuci_logger "Activate conda env"
+rapids-logger "Activate conda env"
+# Permission testing
+set -x
+id
+ls -ln /opt/conda/etc/profile.d/conda.sh
+#
```
Comment (charlesbluca) [inline continuous_integration/gpuci/build.sh:41]:
```suggestion
```

**Review by jrbourbeau — APPROVED** [review]
Thanks @charlesbluca!

**Pull request merged**